extensions/amp-story-dev-tools/0.1/amp-story-dev-tools-tab-logs.js

```diff
@@ -89,14 +104,14 @@ export class AmpStoryDevToolsTabLogs extends AMP.BaseElement {
         this.validateUrl_(/* global amp: false */ amp.validator, this.storyUrl_)
       )
       .then((errorList) => {
-        this.errorList_ = errorList;
+        this.errorList_ = [];
```

Comment:
I think this borks it. Maybe it was for debugging?

Looks good to me after reverting this.

Reply:
Yes! That's for testing for the meantime. Will revert before merging